### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by IngDavidTM**
This is the fourth milestone in the process of creating my portfolio website. It is the desktop version based on Figma design template 3 that uses flexbox and Grid to place elements in the page and forms to fill out and submit. Also, it uses media queries to create diferent layouts depending on the size of the screen.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -111,7 +116,7 @@ <h3 class="finaltext"> I can help you build a product , feature or website Look
             <a class="cv" href="">Get My Resume</a>
         </div>
         <div class="ilustrationme">
-            <article class="cardme">
+            <article class="cardme cm1">
                 <img src="skillsFrame/icon-languages.svg" alt="Languages">
```
Comment (1GORDON) [inline index.html:121]: Great job @IngDavidTM 🤝

- [x] Kindly make sure your card images render properly.
render-images

File: index.html [hunk 2]
```diff
@@ -10,6 +10,11 @@
 </head>
 <body>
     <section class="mainpage">
+        <header class="toolbardesk">
+            <a href="#works">Portfolio</a>
+            <a href="#myself">About</a>
+            <a href="#contact">Contact</a>
+        </header>
```
Comment (1GORDON) [inline index.html:17]: Great job @IngDavidTM 🤝

- [ ] Kindly make sure your design is a close replica of the design provided in the Figma template. You may please add that missing button as shown in the snapshot below.
- Your design
Your design
- Figma design
Figma design

File: index.html [hunk 3]
```diff
@@ -111,7 +116,7 @@ <h3 class="finaltext"> I can help you build a product , feature or website Look
             <a class="cv" href="">Get My Resume</a>
```
Comment (1GORDON) [inline index.html:117]: Great design @IngDavidTM 🤝

- [ ] As stated in the project requirements, kindly implement the button interactions of `hover` and `enable` on this `Get My Resume` button.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @IngDavidTM ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [x] Kindly change your project title in the Readme file from `hello-microverse-project` to `Portfolio`.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @IngDavidTM,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**